webprotege-gwt-ui-shared/src/main/java/edu/stanford/bmir/protege/web/shared/form/data/FormFieldDataDto.java

```diff
@@ -4,14 +4,16 @@
 import com.fasterxml.jackson.annotation.JsonSubTypes.Type;
 import com.google.auto.value.AutoValue;
 import com.google.common.annotations.GwtCompatible;
+import com.google.gwt.user.client.rpc.IsSerializable;
 import edu.stanford.bmir.protege.web.shared.form.field.FormFieldDescriptorDto;
 import edu.stanford.bmir.protege.web.shared.pagination.Page;
 
 import javax.annotation.Nonnull;
+import java.io.Serializable;
 
 @AutoValue
 @GwtCompatible(serializable = true)
-public abstract class FormFieldDataDto {
+public abstract class FormFieldDataDto implements IsSerializable, Serializable {
```

Comment:
Serialization for this (and other form objects) is handled with custom field serializers.  Therefore these objects shouldn't implement IsSerializable, Serializable or have a no args constructor.  Not sure what's the intent of these changes